app/config/checkstyle.xml

```diff
@@ -73,7 +73,6 @@ Severity: HARD
         <module name="MemberName" />
         <module name="MethodName" />
         <module name="PackageName" />
-        <module name="ParameterName" />
```

Comment:
Since this is unrelated to the error prone integration, this is not where the problem lies. See the changelist in the commit where ParameterName label has been added

Reply:
@iamareebjamal You mean in the past commits?